package.json

```diff
@@ -1,24 +1,26 @@
 {
-  "name": "@guidesmiths/react-form-builder",
+  "name": "@onebeyond/react-form-builder",
   "version": "3.6.1",
   "description": "React from builder using json schema",
   "author": "Guidesmiths",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/guidesmiths/react-form-builder.git"
+    "url": "git+https://github.com/onebeyond/react-form-builder.git"
   },
   "main": "dist/index.js",
   "module": "dist/index.modern.js",
   "source": "src/index.js",
   "engines": {
-    "node": ">=10"
+    "node": ">=16",
```

Comment:
We have here node 16, maybe we should use 16 in the .nvmrc

Reply:
🤔